src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -83,12 +87,14 @@ newtype PactServerData_ (v :: ChainwebVersionT) (c :: ChainIdT) logger cas
 data SomePactServerData = forall v c logger cas
     . (KnownChainwebVersionSymbol v,
        KnownChainIdSymbol c,
-       PayloadCas cas)
+       PayloadCas cas,
+       Logger logger)
     => SomePactServerData (PactServerData_ v c logger cas)
 
 
 somePactServerData
     :: PayloadCas cas
+    => Logger logger
```

Comment:
I am not sure if the handler is the best place for logging this data. What about adding the logger on the level of pact service. That would make sure that also internal calls would be logged. I don't think we have those currently, but it may be more future proof.

Reply:
hmm, I just noticed that `validateCommand` is called in the handlers. I don't think that's best practice (normally, handlers should not contain any busyness logic, but only handle HTTP endpoint related things.

Under these circumstances and given that we probably want to log before any business logic is applied to the incoming command, this module is the right place to do the logging.